tools/adev-api-extraction/BUILD.bazel

```diff
@@ -0,0 +1,70 @@
+load("@build_bazel_rules_nodejs//:index.bzl", "nodejs_binary")
```

Comment:
Can you use `js_binary`? `nodejs_binary` is the old one and doesn't support ESM

Reply:
I gave this a try, and it tells me that I have to run the `js_binary` with `js_run_binary` and that seems to be a whole can of worms that involves converting `extract_api_to_json` from a rule to a macro. And that probably entails changing the format of the parameters passed into the actual script.

This is all just copy/pasted from the fw repo with the idea that I would make minimal changes to it and it would eventually be refactored to just call the same script from some shared location.

Reply:
You should be able to just call the js_binary like the nodejs_binary. You just need to set `BAZEL_BINDIR` as environment variable.

I don't think, even if it's for copy/pasting, that we should introduce new instances of legacy code that we were removing/replacing in the last weeks/month.

Reply:
Yeah sorry, I'm not trying to demand that we land legacy code, I was more questioning if this copying even made sense if we were reaching the point of needing to diverge the script, but luckily it sounds like that's not actually the case.